mobile.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Poppins&display=swap">
+    <link rel="stylesheet" href="styles.css">
+    <title>Portfolio Project</title>
+  </head>
+<body>
+    <div class="main-container">
+    <header>
+      <nav class="toolbar">
+          <img class="menu" src="images/ic_menu.png">
+          <img class="menu-shape" src="images/image_geometry_1.png">
+      </nav>
+      <section class="headline">
+        <ul>
+          <li class="my-name">Juan<br>Díaz</li>
+          <li class="myname-intro">Hello! I am a software developer! I can<br>
+                                 help you build a product, feature or<br> 
+                                 website. Take a look at my work.<br><br>      
+                                 If you like what you see and have a<br>
+                                 project you need coded, don’t hesitate to<br>
+                                 contact me.</li>
+        </ul>
```

Comment:
- [ ] currently, you have used a *ul* for all elements. Kindly remove it as the ul tag is used to list items 🙏 

- [ ] Kindly, this is the main title of your page, It would be better to use a ````h1```` tag not a ```li``` tag 👍😃. If you want that google (and other search engines) to know about a specific topic, in this case, you, use an ```h1``` tag. [Check this link](https://www.reliablesoft.net/h1-tag/) for more info. 😎😎😎
- [ ] For the page introduction, it would be better also to use a *p* tag for the best semantic